boards/arm/lpcxpresso55s36/lpcxpresso55s36-pinctrl.dtsi

```diff
@@ -2,7 +2,7 @@
  * NOTE: File generated by lpc_cfg_utils.py
  * from LPC55S36.mex
  *
- * Copyright 2022 NXP
+ * Copyright 2022-2023 NXP
```

Comment:
Please update the board documentation file to indicate support for this feature.